src/back/space/src/service/service.cpp

```diff
@@ -61,37 +61,45 @@ PagingResult<model::Space> Service::GetList(const std::string& userId, uint32_t
 	if (!_defaultSpace.empty()) {
 		const auto defSpace = _repo.Space().SelectByKey(_defaultSpace);
 		if (!_repo.SpaceUser().IsUserInside(defSpace.id, userId)) {
-			_repo.SpaceUser().Insert(defSpace.id, userId, false, Role::Type::User);
+			_repo.SpaceUser().Create(defSpace.id, userId, false, Role::Type::User);
 		}
 	}
-	return _repo.SelectByUserId(userId, start, limit);
+
+	auto data = _repo.SelectByUserId(userId, start, limit);
+	return data;
 }
 
 PagingResult<model::Space> Service::GetAvailableList(const std::string& userId, uint32_t start, uint32_t limit)
 {
-	return _repo.SelectAvailable(userId, start, limit);
+	auto data = _repo.SelectAvailable(userId, start, limit);
+	return data;
 }
 
 PagingResult<model::Space> Service::GetAvailableListBySpaceName(const std::string& spaceName, const std::string& userId, uint32_t start, uint32_t limit)
 {
-	return _repo.SelectAvailableBySpaceName(spaceName, userId, start, limit);
+	auto data = _repo.SelectAvailableBySpaceName(spaceName, userId, start, limit);
+	return data;
 }
 
 PagingResult<model::SpaceInvitation> Service::GetInvitationList(uint32_t start, uint32_t limit, const std::string& userId)
 {
-	return _repo.SelectInvitations(userId, start, limit);
+	auto data = _repo.SelectInvitations(userId, start, limit);
+	return data;
 }
 
 PagingResult<model::SpaceInvitation> Service::GetInvitationListBySpace(const boost::uuids::uuid& spaceId, uint32_t start, uint32_t limit, const std::string& userId)
 {
 	if (!_repo.SpaceUser().IsAdmin(spaceId, userId))
 		throw errors::Forbidden403();
-	return _repo.SelectInvitationsBySpace(spaceId, userId, start, limit);
+
+	auto data = _repo.SelectInvitationsBySpace(spaceId, userId, start, limit);
+	return data;
```

Comment:
А для чего такое изменение? Во многих местах есть. (
```suggestion
	return _repo.SelectInvitationsBySpace(spaceId, userId, start, limit);
```

Reply:
Поправил везде, почему-то думал, что так нагляднее)